config/routes.rb

```diff
@@ -1,5 +1,17 @@
 Rails.application.routes.draw do
-  root 'homes#index'
+  root to: 'homes#index'
```

Comment:
theme is integrated now so change root path to products_path and remove routes and code related to home controller